packages/playground/src/app/Header/index.tsx

```diff
@@ -1,3 +1,55 @@
-export const Header = () => {
-  return <h2 className="m-4">Header</h2>;
+import { ReactElement } from 'react';
+import { Logo } from '~/symbols';
+import { CompileIcon, DownloadIcon, GithubRepoIcon, SettingsIcon } from '~/symbols';
+import { ButtonWithIcon } from '@paritytech/components/ButtonWithIcon';
+
+const openRepoUrl = (): void => {
+  const repoURL = 'https://github.com/paritytech/ink-playground';
+  window.open(repoURL, '_blank');
+};
+
+export const Header = (): ReactElement => {
+  return (
+    <div className="header">
+      <Logo className="h-16 w-32" data-testid="headerLogo" />
+      <div className="verticalDivider" />
+      <div className={'flex p-3.5 w-full'}>
+        <ButtonWithIcon
+          label="Compile"
+          Icon={CompileIcon}
+          testId={'buttonIcon'}
+          onClick={() => {
+            alert('Compile!');
```

Comment:
yesterday the async action PR was merged. We could use the real API call here already.

Reply:
Oh cool, yes, I'll do that.